.gitignore

```diff
@@ -1,5 +1,6 @@
 .idea
 .DS_Store
+.nvmrc
```

Comment:
I added `.nvmrc`  to gitignore because I was trying to test with node 12 and 14 since the CI builds were failing with 14.7.0 -- but only ubuntu. I could not replicate the typescript type issues locally. I can delete if desired.

Reply:
🥳 I rebased main and now was able to reproduce the CI typescript build errors so I could diagnose and rework the errors in ParseJWT. Uses some "any" to get around my minimal understanding of TS at moment.

And all is green!